fibonacci/py/code.py

```diff
@@ -3,12 +3,8 @@
 def fibonacci(n):
   if (n == 0):
     return 0
-  if (n == 1):
-    return 1
-  return fibonacci(n-1) + fibonacci(n-2)
+  return 1 if (n == 1) else fibonacci(n-1) + fibonacci(n-2)
 
 u = int(sys.argv[1])
-r = 0
-for i in range(u):
-  r += fibonacci(i)
+r = sum(fibonacci(i) for i in range(u))
```

Comment:
why are you changing this? your change makes the python code semantically different to all samples in the other languages.

Reply:
Just ran a linter that made those changes. Will remove the changes made to py and js files. Sorry